p2p/rlpx/rlpx.go

```diff
@@ -621,16 +621,16 @@ func (h *handshakeState) readMsg(msg interface{}, prv *ecdsa.PrivateKey, r io.Re
 	return h.rbuf.data[:len(prefix)+len(packet)], err
 }
 
-// sealEIP8 encrypts a handshake message.
-func (h *handshakeState) sealEIP8(msg interface{}) ([]byte, error) {
+// seal encrypts a handshake message.
+func (h *handshakeState) seal(msg interface{}) ([]byte, error) {
 	h.wbuf.reset()
 
 	// Write the message plaintext.
 	if err := rlp.Encode(&h.wbuf, msg); err != nil {
 		return nil, err
 	}
 	// Pad with random amount of data. the amount needs to be at least 100 bytes to make
-	// the message distinguishable from pre-EIP-8 handshakes.
+	// the message distinguishable from pre handshakes.
 	h.wbuf.appendZero(mrand.Intn(100) + 100)
```

Comment:
Seems like we can get rid of this random byte buffer as we don't have pre-EIP-8 handshakes.

Reply:
Removing code that normally functions (no fork switch cases here) seems like it would be out of the scope of this PR